include/boost/dll/detail/demangling/itanium.hpp

```diff
@@ -88,6 +88,29 @@ class mangled_storage_impl : public mangled_storage_base
 
 namespace parser
 {
+    //! declare
+    template <typename... T>
+    struct dummy;
+
+    template <typename T>
+    std::string get_type(const mangled_storage_impl & ms, dummy<T>*);
+
+    template <typename... T, template <typename...> typename Tn>
+    std::string get_type(const mangled_storage_impl & ms, dummy<Tn<T...>>*);
+
+    template <typename... T, template <typename...> typename Tn>
```

Comment:
Use `template <typename...> class ` instead of `template <typename...> typename ` to fix the warning https://travis-ci.com/boostorg/dll/jobs/262855135#L1633